.github/PULL_REQUEST_TEMPLATE

```diff
@@ -6,18 +6,8 @@ A longer description and justification for the changes.
 
 Raise any areas of concern.
 
+A comment will appear below when the live site is built. This normally
+takes a few minutes.
```

Comment:
10 to 15

Reply:
No, it takes a few minutes for the comment to appear. The 10 to 15 minutes is for it to finish the tests. The uploaders don't care about that so much.

Reply:
the comment appears in about 3 minutes